packages/matter-node.js/src/storage/StorageBackendDisk.ts

```diff
@@ -57,4 +57,32 @@ export class StorageBackendDisk implements Storage {
         if (!contexts.length || !key.length) throw new StorageError("Context and key must not be empty strings!");
         this.localStorage.removeItem(this.buildStorageKey(contexts, key));
     }
+
+    /** Returns all keys of a storage context without keys of sub-contexts */
+    keys(contexts: string[]): string[] {
+        if (!contexts.length) throw new StorageError("Context must not be an empty string!");
+        const contextKey = contexts.join(".");
+        if (contextKey.includes("..") || contextKey.startsWith(".") || contextKey.endsWith("."))
+            throw new StorageError("Context must not contain dots!");
```

Comment:
Could we get a helper method for validating the key so we don't need to copy these 4 lines?

Reply:
done